huaweicloud/services/dli/resource_huaweicloud_dli_flinksql_job.go

```diff
@@ -516,6 +574,8 @@ func updateFlinkSqlJobWithStop(ctx context.Context, client *golangsdk.ServiceCli
 	d *schema.ResourceData) diag.Diagnostics {
 
 	if d.HasChangesExcept("smn_topic", "restart_when_exception", "resume_checkpoint", "resume_max_num", "obs_bucket") {
+		log.Print(3333)
```

Comment:
This print thing is using for local debug?

Reply:
fixed